### DISCUSSION TITLE
Make this my current update set not refreshing

### DISCUSSION THREAD

**Discussion by markdangio** [arnoudkooi/SN-Utils#441] [General]
Create two update sets called test1 and click Submit and Make Current. Notice update set picker doesn’t update.

Additionally, create a second update set called test2 and click Submit. Click the related link called: Make this my current update set. Notice the update set picker does not refresh either.

**Comment by markdangio** [comment]
Occurs in ui16 and polaris

**Comment by arnoudkooi** [comment]
It does after a page refresh, doesn't it? It's not easy to watch every event...

**Comment by markdangio** [comment]
Correct it does change after you refresh thru browser. But the dropdown changes without page refresh in OOTB servicenow without sn utils enabledOn Oct 4, 2023, at 1:40 PM, arnoudkooi ***@***.***> wrote:﻿
It does after a page refresh, doesn't it? It's not easy to watch every event...

**Comment by arnoudkooi** [comment]
In UI16? I think you then should disable this setting (be sure to refresh the page for setting to take affect.)

![Google Chrome20231004 - 1958](https://github.com/arnoudkooi/ServiceNow-Utils/assets/840781/f778ccd9-41e7-4772-a471-f7a67897c5b0)

**Comment by markdangio** [comment]
That did the trick for UI 16. Thank you! I am still seeing the issue persist in polaris. Seems to be intermittent.On Oct 4, 2023, at 2:04 PM, arnoudkooi ***@***.***> wrote:﻿
In UI16? I think you then should disable this setting (be sure to refresh the page for setting to take affect.)

**Comment by arnoudkooi** [comment]
Switching the setting off will not show it at all in Next Experience, unless you have the Vancouver Instance tools enabled.
(With the Vancouver Instance Tools, the update didn't seem to happen for me either)

If you want the OOB experience, I would recommend keeping the setting disabled, wouldn't really no what to do else.

**Comment by markdangio** [comment]
Gotcha so is there just no way for snUtils to automatically update the update set name without page refresh in polaris?On Oct 4, 2023, at 3:32 PM, arnoudkooi ***@***.***> wrote:﻿
Switching the setting off will not show it at all, unless you have the Vancouver Instance tools enabled.
(With the Vancouver Instance Tools, the update didn't seem to happen for me either)
If you want the OOB experience, I would recommend keeping the setting disabled, wouldn't really no what to do else.

**Comment by arnoudkooi** [comment]
I'll never say there is now way ;)
I'lll transfer this to an idea, and will update when I find some time to dive in

**Reply by markdangio** [reply]
Thank you so helpful!
